pulsar/core/sippy/SipCiscoGUID.py

```diff
@@ -39,8 +39,8 @@ def __init__(self, body = None, ciscoGUID = None):
         if ciscoGUID != None:
             self.ciscoGUID = ciscoGUID
         else:
-            s = md5(str((random() * 1000000000L) + time())).hexdigest()
-            self.ciscoGUID = (long(s[0:8], 16), long(s[8:16], 16), long(s[16:24], 16), long(s[24:32], 16))
+            s = md5(str((random() * 1000000000) + time())).hexdigest()
+            self.ciscoGUID = (int(s[0:8], 16), int(s[8:16], 16), int(s[16:24], 16), int(s[24:32], 16))
```

Comment:
I am not sure about this. Was this change introduced automatically?

Reply:
I'm not the author, but it was probably introduce automatically, since [int and long got unified in py3](https://www.python.org/dev/peps/pep-0237/).